MovieApp/PresentationLayer/MovieDetails/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+
+class MovieDetailsViewController: UIViewController {
+
+    var navigationView: NavBarView!
+    var headerView: MovieHeaderView!
+
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        .lightContent
+    }
+
+    private let presenter: MovieDetailsPresenter
+
+    init(presenter: MovieDetailsPresenter) {
+        self.presenter = presenter
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        buildViews()
+        loadData()
+    }
+
+    func loadData() {
+        presenter.getMovieDetails(for: 100) { [weak self] result in
```

Comment:
Nope. Movie id should be stored in the presenter and the VC should call just `presenter.getMovieDetails() { [weak self] result in ...`